certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -723,41 +724,62 @@ def _make_server_ssl(self, vhost: obj.VirtualHost) -> None:
 
         """
         https_port = self.config.https_port
-        ipv6info = self.ipv6_info(str(https_port))
-        ipv6_block = ['']
-        ipv4_block = ['']
+        http_port = self.config.http01_port
+
+        # no addresses should have ssl turned on here
+        assert not vhost.ssl
+
+        sslified_addrs: List[obj.Addr] = [obj.Addr(addr.get_addr(), str(https_port), True, False,
```

Comment:
could also do this as `obj.Addr.fromstring(f'{addr.get_addr()}:{https_port} ssl')` if that's more readable

Reply:
+1